public/sass/pages/_explore.scss

```diff
@@ -61,6 +61,14 @@
     padding-left: 0;
     margin-right: auto;
   }
+
+  .time-picker-long-date {
+    .select-button {
+      span {
+        display: none;
+      }
+    }
+  }
```

Comment:
The TimePicker is a generic component that is used from Dashboard (and potentially other places) as well, this should be in _TimePicker.scss

Reply:
Moved to _TimePicker.scss.